Lib/test/test_gc.py

```diff
@@ -877,18 +877,18 @@ def getstats():
         # Z() prevents anything from being collected.
```

Comment:
This, and the comment block above it, are no longer true.  Change to say it _used_ to be true, but that Z() no longer prevents collecting the A()s.